config/routes.rb

```diff
@@ -195,6 +194,11 @@
           put :menu_links, to: 'communities#update_menu_links'
           delete :delete_marketplace
 
+          # DEPRECATED (2016-07-07)
+          # These routes are not in use anymore, don't use them
+          # See new "Guide" routes above, outside of communities resource
+          get :getting_started, to: 'getting_started_guide#index'
```

Comment:
I guess redirect would be more appropriate than routing the old URL to new controller?